File: backend/utils/payrollCalculator.js
export function calculatePayroll(basic, allowances = 0, deductions = 0) {
  const gross = basic + allowances;
  const paye = gross * 0.05;     // Sample PAYE % for now
  const nhif = 500;
  const nssf = 200;
  const net = gross - (paye + nhif + nssf + deductions);
  return { gross, paye, nhif, nssf, net };
}

File: backend/utils/payrollCalculator.py
def calculate_payroll(basic, allowances=0, deductions=0):
    gross = basic + allowances
    paye = gross * 0.05
    nhif = 500
    nssf = 200
    net = gross - (paye + nhif + nssf + deductions)
    return {"gross":gross,"paye":paye,"nhif":nhif,"nssf":nssf,"net":net}
